index.html

```diff
@@ -2,15 +2,27 @@
 layout: default
 ---
 
-<div class="block">
-  {% include before.html %}
-</div>
-<div class="block">
-  <h1><span class="header-inside">Projects</span></h1>
-  <div id="projects-panel">
-    {% include noscript.html %}
+<div class="card">
+
+  <p class="name">Syed Arfath</p>
+  <p class="contact">www.linkedin.com/in/arfath-syed-387511189</p>
+  <p class="about">I am practising on my github profile and learning how to merge it</p>
+  </div class="resources">
+    <p>3 Useful dev resources are</p>
+    <ul>
+      <li>
+        <a href="https://www.youtube.com/c/KunalKushwaha" target="_blank" title="This is the java resource">The java resource</a>
+      </li>
+
+      <li>
+        <a href="https://www.youtube.com/c/AdityaVermaTheProgrammingLord" target="_blank" title="This is the second resource">The resource to recursion</a>
+      </li>
+
+      <li>
+        <a href="https://www.youtube.com/c/TechWithTim" target="_blank" title="This is the third resource">The python resource</a>
+      </li>
+    </ul>
```

Comment:
@arfath99 I am not sure what were you trying to do with this. If you meant something else. Please let us know and would be happy to help you. Closing this out being inappropriate.